vendor/processors/stripe_processor.py

```diff
@@ -462,29 +470,41 @@ def create_stripe_customers(self, customers):
         for profile in customers:
             profile_data = self.build_customer(profile)
             new_stripe_customer = self.stripe_create_object(self.stripe.Customer, profile_data)
-
+            print(f'building new customer off {profile_data}')
             if new_stripe_customer:
                 profile.meta['stripe_id'] = new_stripe_customer['id']
                 profile.save()
 
     def update_stripe_customers(self, customers):
         for profile in customers:
-            customer_id = profile.meta['stripe_id']
-            profile_data = self.build_customer(profile)
-            existing_stripe_customer = self.stripe_update_object(self.stripe.Customer, customer_id, profile_data)
+            customer_id = profile.meta.get('stripe_id')
+            print(f'checking on customer {profile} and meta {profile.meta}')
+            if customer_id:
```

Comment:
Why are you checking for a `stripe_id` if that function is suppose to reviece customer that do have a `stripe_id`?